cypress/integration/pages/topicPage/tests.js

```diff
@@ -0,0 +1,116 @@
+import envConfig from '../../../support/config/envs';
+
+export default ({ service, pageType }) => {
+  describe(`Tests for ${service} ${pageType}`, () => {
+    let topicId;
+    let firstItemHeadline;
```

Comment:
this variable could be deleted and be a const on line 80 because it never changes

Reply:
It seems better to have these in the upper scope outside the it blocks so all the blocks can access the variable without redeclaring it in each test.

Reply:
oh yeh you're tight. didn't see that!